src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseAssignedUserRolesController.java

```diff
@@ -242,6 +244,54 @@ public ResponseEntity<CaseAssignedUserRolesResource> getCaseUserRoles(@RequestPa
         return ResponseEntity.ok(new CaseAssignedUserRolesResource(caseAssignedUserRoles));
     }
 
+    @PostMapping(
+        path = "/case-users/search"
+    )
+    @ApiOperation(
+        value = "Get Case-Assigned Users and Roles"
```

Comment:
This is essentially the same operation as the original `GET`, and is the long-term replacement.